wormhole/recv.go

```diff
@@ -273,6 +273,13 @@ func (c *Client) Receive(ctx context.Context, code string, disableListener bool,
 		return nil
 	}
 
+	go func() {
```

Comment:
What is idea behind this goroutine? To close encrypted connection with pair or relay from receiver side when context is finished?

Reply:
Yes this closes the connection of an ongoing transfer that's created when the receiver accepts:

https://github.com/LeastAuthority/wormhole-william/blob/5c52629f26e98810f57b6a356fbba2fa5f955f9a/wormhole/recv.go#L269-L271

Right now if the receiver cancels the context on the wasm side, the receiver exits fine but the sender keeps going because the connection is still open:

https://github.com/LeastAuthority/wormhole-william/blob/5c52629f26e98810f57b6a356fbba2fa5f955f9a/wasm/client.go#L342-L345

So this goroutine ensures the connection is closed on cancel

Reply:
Ok. Sender side does the same, only it close connection directly not via cryptor function: https://github.com/LeastAuthority/wormhole-william/blob/10940cd31c7445ede9561db3ef08f566d95b5f3e/wormhole/send.go#L412